test/selectors-relative.spec.js

```diff
@@ -1,33 +1,38 @@
-const Promise = require("promise");
-const express = require("express");
-const { firefox } = require("playwright");
+const fs = require("fs");
+const process = require("process");
+const { chromium } = require("playwright");
 const relativeSelector = require("../src/index");
-let server, port;
+
+const headless =
```

Comment:
This seems a bit overcomplicated

Reply:
Initially I want to avoid to add 'crossenv' to limit the number of dependencies.
But here I have the value 'true  ' with extra space 🙃
Pheraps I should reconsider crossenv